rivers/yahoo-finance-stats/parser.js

```diff
@@ -0,0 +1,41 @@
+var _ = require('lodash'),
+    moment = require('moment-timezone');
+
+module.exports = function(config, body, url, temporalDataCallback, metaDataCallback) {
+    var data = JSON.parse(body);
+
+    // This is important.
+    moment.tz.setDefault("UTC");
+
+    var dtarray = data.query.created.split('T'),
+        dateString = dtarray.shift(),
+        timeString = dtarray.shift().split('Z').shift(),
+        date = moment(dateString + ' ' + timeString, 'YYYY-MM-DD HH:mm:ss'),
+        timestamp = date.unix();
+
+    var res = data.query.results.quote,
+        symbol = res.symbol,
+        EBITDAStr = res.EBITDA,
+        fieldValues;
+
+    if (EBITDAStr.indexOf('M') != -1) {
+        var EBDITA = parseFloat(EBITDAStr.split("M").shift())*1000000;
+    } else if (EBITDAStr.indexOf('B') != -1) {
+        var EBDITA = parseFloat(EBITDAStr.split("B").shift())*1000000000;
+    } else {
+        var EBDITA = parseFloat(EBITDAStr)
+    }
```

Comment:
It is bad karma to declare vars in many places like this. Declare it at the top of the function scope as `undefined`, then assign it later.